Item23_Understand_std_move_and_std_forward/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-set_source_files_properties(std_move_in_cpp14.cpp PROPERTIES COMPILE_FLAGS "-std=c++14 -stdlib=libc++")
+set_source_files_properties(std_move_in_cpp14.cpp PROPERTIES COMPILE_FLAGS "-std=c++14")
```

Comment:
Would it be possible for you to make sure everything builds and compiles with both clang and g++ instead of having separate branches for each of the compiler versions?  I'm quite sure CMake allows you to check what compiler is used and add options depending on compiler.  Feel free to do this and send me the pull request :-)